### PR TITLE
Upstream Hotfix pipeline

### PR DESCRIPTION
**Pull request by prashantmhetar**
The pipeline to create a RPM build and test the build for non-compatible errata kernel

### CODE REVIEW

**Comment by johnsongeorge-w** [comment]
@iamshital : Review the pipeline

File: linux_pipeline/Jenkinsfile_azure_lis_hotfix_pipeline [hunk 1]
```diff
@@ -0,0 +1,644 @@
...
+
+        currentStage = "Prepare Environment"
+        stage (currentStage) {
+             node ("azure") {
+                try {
+                    def distro = "${HotfixTargetList}"
+                    distro = distro.replace('[', '').replace(']', '').replace(' ', '')
+                    println "Prepare environment for Distro ${distro}"
+                    checkout scm
+                    withCredentials([file(credentialsId: 'Azure_Secrets_File', variable: 'Azure_Secrets_File'), string(credentialsId: 'LIS_BUILD_PASSWORD', variable: 'LIS_BUILD_PASSWORD')])
```
[the remark this answers is not in the document]
Comment (iamshital) [reply linux_pipeline/Jenkinsfile_azure_lis_hotfix_pipeline]: @prashantmhetar , Please change ID to : Azure_Secrets_File_LIS_Val and upload the "Dev & Test" subscription's secret file with this ID for time being. Once our environment is switched to LIS Validation subscription we will overwrite this file. @LiliDeng  Is that Okay?

File: linux_pipeline/Jenkinsfile_azure_lis_hotfix_pipeline [hunk 2]
```diff
@@ -0,0 +1,644 @@
...
+                 "rhel_7.0": "NA", "rhel_7.1": "NA", "rhel_7.2": "RedHat RHEL 7.2 Latest",
+                 "rhel_7.3": "RedHat RHEL 7.3 Latest", "rhel_7.4": "RedHat RHEL 7.4 Latest", "rhel_7.5": "RedHat RHEL 7.5 Latest",
+                 "rhel_7.6": "RedHat RHEL 7-RAW 7.6.2018103108"]
+def HyperVImages = ["rhel_6.5": "rhel_6.5", "rhel_6.6": "rhel_6.6_x64", "rhel_6.7": "NA",
+            "rhel_6.8": "rhel_6.8_x64", "rhel_6.9": "rhel_6.9_x64", "rhel_6.10": "rhel_6.10_x64",
+             "rhel_7.0": "rhel_7.0", "rhel_7.1": "rhel_7.1", "rhel_7.2": "rhel_7.2",
+             "rhel_7.3": "rhel_7.3", "rhel_7.4": "rhel_7.4", "rhel_7.5": "rhel_7.5",
+             "rhel_7.6": "rhel_7.6"]
+
+def HotfixTargetList = TARGET_LIST.split("\n")
```
[the remark this answers is not in the document]
Comment (prashantmhetar) [reply linux_pipeline/Jenkinsfile_azure_lis_hotfix_pipeline]: updated parameter name to DISTRO_KERNEL

File: linux_pipeline/Jenkinsfile_azure_lis_hotfix_pipeline (hunk 1 above)
[the remark this answers is not in the document]
Comment (prashantmhetar) [reply linux_pipeline/Jenkinsfile_azure_lis_hotfix_pipeline]: changed the ID to Azure_Secrets_File_LIS_Val

**Comment by prashantmhetar** [comment]
### **This PR is Verified**

**Pull request merged**